src/main/java/com/solopov/urlshorter/web/dto/UrlGenerateRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package com.solopov.urlshorter.web.dto;
+
+import com.solopov.urlshorter.dao.entities.ShortenedUrl;
+
+import javax.validation.constraints.NotBlank;
+
+public record UrlGenerateRequestDto(@NotBlank String url, @NotBlank String title) {
```

Comment:
exceptions for violated constraints aren't handled which results into 500s

Reply:
upd commit: added controller advice